ops/test/_config.tf

```diff
@@ -10,10 +10,11 @@ terraform {
 terraform {
   required_providers {
     azurerm = {
-      source = "hashicorp/azurerm"
+      source  = "hashicorp/azurerm"
+      version = "~> 2.93"
     }
   }
-  required_version = ">= 0.15.1"
+  required_version = ">=1.0.11"
```

Comment:
I think this needs to be `1.1.4`.

Reply:
Find and replace failed me - good catch! Just pushed a fix